pyimzml/ImzMLWriter.py

```diff
@@ -105,16 +106,25 @@
 """
 
 class ImzMLWriter(object):
-    def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.float32):
+    def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.float32, mz_compression=False, intensity_compression=False):
+        '''"output_filename" is used to make the base name by removing the extension (if any).
+        two files will be made by adding ".ibd" and ".imzML" to the base name
+        processed or continuous mode will be chosen automatically. Identical mz lists will only be written once.
+        Force process mode by setting "instance.hashes = None"
+        "compression" must be True or False or indicate a number to round to.
```

Comment:
This looks rather cumbersome, and it's possible that more compression options will be added to the standard later.

I suggest to introduce two classes for clarity:
*  `NoCompression`
*  `ZlibCompression` accepting a constructor parameter `round=None|#decimals`